execution/executor-benchmark/src/lib.rs

```diff
@@ -35,7 +35,6 @@ use std::{
     path::PathBuf,
     sync::{mpsc, Arc},
 };
-use storage_client::StorageClient;
```

Comment:
why?

Reply:
it's too slow, 30% performance drop using this client